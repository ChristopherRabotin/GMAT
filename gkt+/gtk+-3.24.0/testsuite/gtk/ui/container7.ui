<!-- invalid container packing attribute -->
<interface>
  <object class="GtkBox">
    <child>
      <object class="GtkLabel" id="widget1"/>
      <packing try="this">
      </packing>
    </child>
  </object>
</interface>
