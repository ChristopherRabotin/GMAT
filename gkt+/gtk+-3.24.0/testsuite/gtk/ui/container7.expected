ERROR: g-markup-error-quark 4
.:7:1 attribute 'try' invalid for element 'packing'
